pkgs/development/python-modules/brotlicffi/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, cffi
+, brotli
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "brotlicffi";
+  version = "1.0.9.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "python-hyper";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0qx7an7772brmx1rbbrqzqnkqqvicc70mx740nl31kzzyv4jjs00";
+  };
+
+  preBuild = ''
+    export USE_SHARED_BROTLI=1
+  '';
+
+  propagatedBuildInputs = [
+    brotli
```

Comment:
Shouldn't something be linked against this which means we do not need to propagate it?

Reply:
Indeed, updated.

Reply:
You didn't push a change.

Reply:
Again, thanks.